js/components/Table/Row.js

```diff
@@ -11,7 +11,7 @@ class Row extends React.Component {
   createRow(data) {
     const row = [
       {
-        title: data.user.data.first_name,
+        title: data.rank.toString().concat('. ', data.user.data.first_name),
```

Comment:
I might have not done it the best way and I haven't pushed up a PR yet, but the way I did the images was for them to be in a column to the left of the name. This looks good except the way I did images won't work with it so I might need help figuring that out

Reply:
@katiecrane ok! Happy to 👀 !